lib_test/test_str.ml

```diff
@@ -6,8 +6,6 @@ let eq_match ?pos ?len ?(case = true) r s =
     ~printer:arr_ofs_printer
     (fun () ->
        let pos = match pos with None -> 0 | Some p -> p in
-       let pat = if case then Str.regexp r else Str.regexp_case_fold r in
-       let _s_start = Str.search_forward pat s pos in
```

Comment:
Pretty sure this isn't semantically equivalent however. `search_forward` raises when a result isn't found.

So I guess `ignore (Str.search_forward ..)` is equivalent. What do you think?